test/macros_test.exs

```diff
@@ -96,4 +96,39 @@ defmodule Bureaucrat.MacrosTest do
       assert error.message =~ "Bureaucrat couldn't find a controller and/or action for this request"
     end
   end
+
+  # We want to raise an error when Phoenix.ConnTest macros are called outside a `test` block.
+  # This test mimics a userland test that calls Phoenix.ConnTest.{get, post, put, delete}
+  # and asserts that a RuntimeError is raised when compiling the code.
+  # This is counterintuitive, but remember that some of Bureaucrat's runtime is during test compilation.
+  for method <- [:get, :post, :put, :delete] do
+    @method method
```

Comment:
I don't think you need this. You could reference `method` directly.

Reply:
It is needed. I think it's so that the module is available at compilation time for line 107 and 124.